pkg/actions/lua/lakefs/catalogexport/glue_exporter.lua

```diff
@@ -143,6 +153,6 @@ local function export_glue(glue, db, table_src_path, create_table_input, action_
 end
 
 return {
-    get_full_table_name=get_full_table_name,
+    get_full_table_name = get_full_table_name,
```

Comment:
What is this return thing outside of any function? A mistake? Some Lua thing?

Reply:
@Isan-Rivkin what is the story with this return command at the end of glu_exporter.lua that exists outside any function? Is this a mistake? Does it do something?